ci-operator/step-registry/gcp/OWNERS

```diff
@@ -1,4 +1,6 @@
 approvers:
+- wgahnagl
```

Comment:
I meant you can create a plain OWNER file under `gcp/{provision,deprovision}/buildhost/`,  then place your team member as owners for the newly created steps, but not touch with parent folder's OWNER list.